### PR TITLE
Fix channel rep count

### LINKED ISSUE #2628: On community views, users show global rep vs local rep.
image

### PR DESCRIPTION
**Pull request by brianlovin**
**Status**
- [ ] WIP
- [x] Ready for review
- [ ] Needs testing

**Deploy after merge (delete what needn't be deployed)**
- hyperion (frontend)

**Release notes for users (delete if codebase-only change)**
- Fixes a bug where the wrong rep score would show up when viewing channel members

Closes #2628 

Note: most of this will need a refactor when we update the `Channel.membersConnection` api to be more in sync with the way we structure `Community.members`

### CODE REVIEW

File: api/mutations/thread/publishThread.js [hunk 1]
```diff
@@ -104,26 +104,26 @@ export default async (
...
+  // const hasFreePrivateChannel = await StripeUtil.hasSubscriptionItemOfType(
+  //   customer,
+  //   FREE_PRIVATE_CHANNEL
+  // );
+
+  // if (channel.isPrivate && (!hasPaidPrivateChannel && !hasFreePrivateChannel)) {
+  //   return new UserError(
+  //     'This private channel does not have an active subscription'
+  //   );
+  // }
```
Comment (mxstbr) [inline api/mutations/thread/publishThread.js]: What is all of this commented code?

Comment (brianlovin) [reply api/mutations/thread/publishThread.js]: Whoops!

Comment (brianlovin) [reply api/mutations/thread/publishThread.js]: Its commented because the local stripe setup breaks thread publishing if I'm running off a db backup :P

**Review by mxstbr — APPROVED** [review]
LGTM!

**Pull request merged**